server/datastore/mysql/migrations/tables/20210708143152_AddHostUsers.go

```diff
@@ -0,0 +1,33 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20210708143152, Down_20210708143152)
+}
+
+func Up_20210708143152(tx *sql.Tx) error {
+	sqlStatement := `
+		CREATE TABLE IF NOT EXISTS host_users (
+		id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+		host_id INT UNSIGNED NOT NULL,
+		uid INT UNSIGNED NOT NULL,
+		username VARCHAR(255),
+		groupname VARCHAR(255),
+		created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+		user_type VARCHAR(255),
+		UNIQUE KEY idx_uid_username (uid, username),
```

Comment:
I don't think this is an acceptable unique key, because `(0, root)` will be on many systems. Perhaps test should be updated to include multiple hosts wit hthe same `uid, username` pairs?

Reply:
You're right! My mind was thinking only at the host level. I'll add host_id there.